icu_transform.c

```diff
@@ -153,7 +178,6 @@ icu_transform(PG_FUNCTION_ARGS)
 		elog(ERROR, "utrans_transUChars failed: %s", u_errorName(status));
```

Comment:
This call should be removed too, or alternatively utrans should be set to NULL after utrans_close().